source/_posts/2016-11-23-alan-and-grace-an-origin-story.markdown

```diff
@@ -0,0 +1,92 @@
+imagination of Alan Turing. Cue the mic drop now.
+
+<div align="center">
+{% img left /assets/2016-11-23-alan-and-grace-an-origin-story/alan-turing.jpg Alan %}
+{% img right /assets/2016-11-23-alan-and-grace-an-origin-story/turing-machine.jpg Turing Machine %}
+
+<sub>Sources: <a href="http://www.alanturing.net/turing_archive/graphics/photos%20of%20Turing/pages/alan1_psd.htm">alanturing.net</a>, <a href="http://worldnews.nbcnews.com/_news/2013/12/23/22025978-queen-pardons-computing-giant-alan-turing-59-years-after-his-suicide">nbcnews.com</a></sub>
+</div>
+
+___
+
+{% img center /assets/2016-11-23-alan-and-grace-an-origin-story/grace-8bit.png Grace %}
+
+Cerner Engineering’s jovial female mascot appears as if she’s ready to take on
+the world. The fearless spirit of the mascot encompases the nature of its muse
+for whom she was named, Grace Brewster Murray Hopper (1906-1992). Grace Hopper
+was an American mathematician, computer programmer, military leader, and a woman
+clearly ahead of her time. Grace received a Ph.D. in mathematics, one of the
+first women to earn this level of education. Grace’s path to computer science
+began while serving in the U.S. Naval Reserve as one of the first to program the
+Harvard Mark I computer. Grace went on to create the first compiler for
+computer languages leading to COBOL, a widely adapted language that would be
+used around the world. But Grace wasn’t finished yet. At the age of 60, she
+was recalled to active duty where she standardized communication between
+different computer languages. When Grace retired at the age of 79 as a rear
+admiral, she was the oldest serving officer in the service. In 1991, Grace was
+the first women awarded the National Medal of Technology. Throughout her life,
+Grace encouraged young people to learn how to program. Grace’s legacy lives on
+through the Grace Murray Hopper Award, the Grace Hopper Celebration of Women in
+Computing Conference, the University of Missouri’s computer museum “Grace’s
+Place”, and now Cerner’s Engineering mascot. Grace continues to serve as an
+inspiration to women in tech everywhere. Was there any other choice?
+
+<div align="center">
+{% img left /assets/2016-11-23-alan-and-grace-an-origin-story/grace-hopper.jpg Grace Hopper %}
+{% img right /assets/2016-11-23-alan-and-grace-an-origin-story/grace-hopper-2.jpg Grace Hopper %}
+
+<sub>Sources: <a href="http://www.womenatworkmuseum.org/envision-engineering.html">womenatworkmuseum.org</a>, <a href="https://en.wikipedia.org/wiki/Grace_Hopper">wikipedia.org</a></sub>
+</div>
+
+Alan and Grace represent a beacon of inspiration, providing icons that
+encapsulate and promote a culture of innovation, forward thinking, and just
+plain fun. The creation of mascots Alan and Grace has been a big win for the
+passionate Cerner engineers dedicated to fostering Cerner’s Engineering culture.
+Alan and Grace represent a beacon of inspiration, providing icons that
```

Comment:
I just removed that duplicate sentence typo in 6053ed6 and deployed it in 5fcd0b